Functions.Templates/Templates/BlobTrigger-PowerShell/function.json

```diff
@@ -0,0 +1,12 @@
+{
+    "disabled": false,
+    "bindings": [
+        {
+            "name": "Blob",

```

Comment:
InputBlob? or BlobItem?

Reply:
The type of the input trigger biding is `blobTrigger`, so I guess using `Blob` as the name is not confusing. What do you think?

Reply:
That makes sense. I will change it to `InputBlob`